pkg/util/config.go

```diff
@@ -220,3 +221,14 @@ func DefaultCPUBurstConfig() slov1alpha1.CPUBurstConfig {
 		CFSQuotaBurstPeriodSeconds: pointer.Int64Ptr(-1),
 	}
 }
+
+type NodeCPUBurstCfg struct {
+	// An empty label selector matches all objects while a nil label selector matches no objects.
+	NodeSelector *metav1.LabelSelector `json:"nodeSelector,omitempty"`
+	*slov1alpha1.CPUBurstStrategy
+}
+
+type CPUBurstCfg struct {
```

Comment:
These definitions are redudant with `CPUBurstCfg`, `NodeCPUBurstCfg` in `pkg/slo-controller/config/config.go`.